Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM ubuntu:disco
+FROM ubuntu:focal
 
 ARG MAKEFLAGS=-j2
+ARG DEBIAN_FRONTEND=noninteractive
 
 ENV GOROOT /opt/go
 ENV PATH ${GOROOT}/bin:${GOPATH}/bin:${PATH}
 ENV NFF_GO /nff-go
 
 RUN apt-get -q update && apt-get -q -y install \
     make \
+    build-essential \
```

Comment:
libstdc++ was missing, making examples/dpi tests fail. It's included in the build-essential.